components/style/style_adjuster.rs

```diff
@@ -557,6 +557,12 @@ impl<'a, 'b: 'a> StyleAdjuster<'a, 'b> {
         layout_parent_style: &ComputedValues,
         flags: CascadeFlags,
     ) {
+        // Don't adjust visited styles, visited-dependent properties aren't
+        // affected by these adjustments and it'd be just wasted work anyway.
```

Comment:
Consider adding a comment mentioning that it is important to skip adjustments, because we don't cascade non-visited-dependent properties for visited styles at all.

Reply:
Well, it's not important, in the sense that we'll never look those properties up anyway. But sure, I'll mention it.

Reply:
It's important in sense of that we have assertion checking that... but yeah, as far as we never look at those properties, they wouldn't really affect the rendering.